frontend/src/metabase/admin/datamodel/components/PartialQueryBuilder.jsx

```diff
@@ -21,7 +21,7 @@ import withTableMetadataLoaded from "../hoc/withTableMetadataLoaded";
 class PartialQueryBuilder extends Component {
   static propTypes = {
     onChange: PropTypes.func.isRequired,
-    table: PropTypes.object.isRequired,
+    table: PropTypes.object,
```

Comment:
The prop value for `table` was being passed as `null` when running the test, so I removed the `required` on this. The test works now, and from what I can see in the code for `PartialQueryBuilder`, we have the checks for `table === falsy` within the code so I think it _should_ be ok